crates/live_update/src/jni.rs

```diff
@@ -15,60 +15,73 @@
 use std::ffi::c_void;
 
 use android_logger::Config;
-use figma_import::{fetch_doc, ConvertRequest, Error};
+use jni::objects::{JByteArray, JObject};
 use jni::objects::{JClass, JString};
-use jni::sys::{jbyteArray, jint, JNI_VERSION_1_6};
+use jni::sys::{jint, JNI_VERSION_1_6};
 use jni::{JNIEnv, JavaVM};
 use log::error;
 use log::LevelFilter;
 
-fn throw_basic_exception(env: JNIEnv, msg: String) {
+use figma_import::{fetch_doc, ConvertRequest};
+
+use crate::error_map::map_err_to_exception;
+
+fn throw_basic_exception(env: &mut JNIEnv, msg: String) {
     //An error occurring while trying to throw an exception shouldn't happen, but let's at least panic with a decent error message
     env.throw(msg).expect("Error while trying to throw Exception");
 }
 
-fn jni_fetch_doc(
-    env: JNIEnv,
+fn jni_fetch_doc<'local>(
+    mut env: JNIEnv<'local>,
     _class: JClass,
     jdoc_id: JString,
     jrequest_json: JString,
-) -> jbyteArray {
-    let doc_id: String = match env.get_string(jdoc_id) {
+) -> JByteArray<'local> {
```

Comment:
Is `jbyteArray` different than `JByteArray`?

Reply:
Yes, it includes lifetime data